Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y curl  libunwind8 libicu55
+RUN curl -LO https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.9/powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb
```

Comment:
This is hard-coded; but totally a-okay for now. We can do more clever stuff later.